doc/AWSLambdaConnector.md

```diff
@@ -63,7 +63,7 @@ _Connector actions_:
 
 [createFromBuffer](#createFromBuffer) Create a new Lambda function from buffer
 
-[createFromCode](#createFromCode) Create a new Lambda function from code
+[createFromCode](#createFromCode) Create a new Lambda function from code string
```

Comment:
Nit: we are sort of overloading the term 'code' in the various actions. In one meaning - it is 'raw' code (like here) and in the other - it is existing code (like in `updateCode` below).
This is not an error but could get confusing. Especially in `updateCodeFromCode`...

As an aside -why do we add these wrapper functions

Reply:
Useful